test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -815,6 +815,15 @@ def forward(self, x, y):
         y = torch.randn(2, 3, 4)
         self.run_test(FloorDivModule(), (x, y))
 
+    def test_floordiv(self):
```

Comment:
Looks good. Please write the test as a nn.Module and enableScript.
Thanks.

Reply:
Also, looks like your test is using ConstantOfShape which is implemented in opset 9. Please set the test to run for opesets 9 and higher.